libs/blocks/path-finder/path-finder.js

```diff
@@ -0,0 +1,103 @@
+import login from '../../tools/sharepoint/login.js';
+import { account } from '../../tools/sharepoint/state.js';
+import getServiceConfig from '../../utils/service-config.js';
+import { getReqOptions } from '../../tools/sharepoint/msal.js';
+import { createTag } from '../../utils/utils.js';
+
+const SCOPES = ['files.readwrite', 'sites.readwrite.all'];
+const TELEMETRY = { application: { appName: 'Milo - Where am I' } };
+
+const getSharePointDetails = (() => {
+  let site;
+  let driveId;
+  let reqOpts;
+
+  return async () => {
+    if (site && driveId && reqOpts) return { site, driveId, reqOpts };
+    console.log('fetching sp details');
```

Comment:
Would you still want to keep this logged?

Reply:
I removed it.